aten/src/ATen/cpu/vec/vec256/vec256_complex_double.h

```diff
@@ -267,6 +267,9 @@ template <> class Vectorized<c10::complex<double>> {
   Vectorized<c10::complex<double>> hypot(const Vectorized<c10::complex<double>> &) const {
     AT_ERROR("not supported for complex numbers");
   }
+  Vectorized<c10::complex<double>> digamma(const Vectorized<c10::complex<double>> &) const {
+    AT_ERROR("not supported for complex numbers");
+  }
```

Comment:
Hi,
I think maybe we should remove this because `digamma` does not support complex inputs. Or is it right to keep it as a warning?

Thanks!

Reply:
I'm also surprised to see similar `AT_ERROR` invocations in the vicinity for other ops that also don't support complex. I see that they have been added since https://github.com/pytorch/pytorch/pull/26500/files#diff-aeacfa0760faa87f15d153cf5bd4fb186caec2117712ae62e12779a845a37f69.

We can remove them, if there wouldn't be any side-effect.

Reply:
Thanks, @sanchitintel, for looking into it. I've removed it in my latest commit.

I see there are many other ops that add `AT_ERROR` for operators that do not support complex inputs. I'll submit another PR, either removing it or adding complex support. Will that be okay? Thanks!

Reply:
SGTM, thanks!